lib/sidekiq/api.rb

```diff
@@ -943,8 +943,27 @@ def leader
   #   'busy' => 10,
   #   'beat' => <last heartbeat>,
   #   'identity' => <unique string identifying the process>,
+  #   'embedded' => true,
   # }
   class Process
+    def self.find(identity)
```

Comment:
I'd prefer to avoid method names that clash with Enumerable, e.g. `ProcessSet.new.find(...)`. How about `Sidekiq::Process.lookup(identity)` or `Sidekiq::ProcessSet[identity]`, i.e. `def self.[](id)`?

Reply:
`Process` isn't `Enumerable` and I prefer to keep names consistent with what a user would expect (e.g. `ActiveRecord` `find` loads a model by id). Your repo though, I'll change to `lookup`.

Reply:
Actually, I prefer `ProcessSet#[]`. Makes sense that you can enumerate or get a single one via the same interface

Reply:
This has been updated